docs/dev/framework/async-messaging.md

```diff
@@ -0,0 +1,298 @@
+minutely cron job triggering `contao:cron` and you're good to go!
+{{% /notice %}}
+
+## The priority message interfaces
+
+So we know how the transport works, and we have a solution for running the `messenger:consume` commands. One piece is 
+missing, though: How does Contao know that your message (let's assume a `CreateAsyncZipFileMessage` in this example) 
+should be routed to the `contao_prio_low`, `contao_prio_normal` or `contao_prio_high` transports? The routing part is missing!
+So as an extension developer, you would need to specify the target like so:
+
+```yaml
+framework:
+    messenger:
+        routing:
+            'App\Messenger\CreateAsyncZipFileMessage': contao_prio_high
+```
+
+This would be totally doable using a `Contao Manager Plugin` and adjusting the Symfony Framework configuration, 
+appending your entry. However, because Contao ships with the 3 default priorities, there are also built-in 
+interfaces for those 3 which are then routed automatically:
+
+```yaml
+framework:
+    messenger:
+        routing:
+            'Contao\CoreBundle\Messenger\Message\HighPriorityMessageInterface': contao_prio_high
+            'Contao\CoreBundle\Messenger\Message\NormalPriorityMessageInterface': contao_prio_normal
+            'Contao\CoreBundle\Messenger\Message\LowPriorityMessageInterface': contao_prio_low
+```
+
+{{% notice idea %}}
+Instead of fiddling with the container and the configuration, all you need to do is implement one of the priority
+interfaces and the routing is configured.
+{{% /notice %}}
+
+## Using the entire Contao Managed Edition framework as a developer
+
+The entire setup presented above ensures that - as a developer - you can enjoy a zero-configuration asynchronous 
+message processing setup, provided you have the Contao cronjob framework running. You only need your message and the 
+respective message handler:
+
+1. Register a minutely cronjob for `contao:cron` - aka configure the Contao cron job framework.
+2. Create your message:
+    ```php
+    namespace App\Messenger;
+    
+    use Contao\CoreBundle\Messenger\Message\HighPriorityMessageInterface;
+   
+    class CreateAsyncZipFileMessage implements HighPriorityMessageInterface
+    {
+        public function __construct(public array $fileIds)
+        {
+        }
+    }
+    ```
+3. Create your message handler:
+    ```php
+    namespace App\Messenger;
+        
+    use Symfony\Component\Messenger\Attribute\AsMessageHandler;
+
+    #[AsMessageHandler]
+    class SearchIndexMessageHandler
+    {
+        public function __invoke(CreateAsyncZipFileMessage $message): void
+        {
+            foreach ($message->fileIds as $fileId) {
+                // Create your zip file asynchronously which can take a long while now 🔥
+            }
+        }
+    }
+    ```
+4. Dispatch the message from e.g. your controller and watch as the magic unfolds.
+5. Done! 🎉
+
```

Comment:
You could add something like this here:

_For a working example, take a look at the `SearchIndexMessage`, `SearchIndexMessageHandler` and `SearchIndexListener` classes to see how Contao uses the messenger to create and update the search index._

But feel free not to. 😄

Reply:
Sure, great idea! Added in https://github.com/contao/docs/pull/1228/commits/227ce0eabbf5c77d3298be27f8ec4c8fdfbece35